features/tests/test_api.py

```diff
@@ -97,6 +98,31 @@ def test_features_query(snapshot, api_client):
     snapshot.assert_match(executed)
 
 
+@freeze_time("2019-12-16 12:00:01")
```

Comment:
You can remove the `freeze_time` since since timestamps are not involved in this test.